indico/cli/admin.py

```diff
@@ -37,13 +40,66 @@ def cli():
 def _print_user_info(user):
     print()
     print('User info:')
+    print("  ID: {}".format(user.id))
     print("  First name: {}".format(user.first_name))
     print("  Family name: {}".format(user.last_name))
     print("  Email: {}".format(user.email))
     print("  Affiliation: {}".format(user.affiliation))
     print()
 
 
+def _safe_lower(s):
+    return (s or '').lower()
+
+
+@cli.command()
+@click.option('--substring', is_flag=True,
+              help='Whether to require exact matches (the default) or do substring matches (slower)')
```

Comment:
isn't it more common to see `default` instead of `the default`? Usually the latter is followed by the word `value`.